instrumentation-core/spring/spring-webmvc-3.1-core/spring-webmvc-3.1-core.gradle

```diff
@@ -0,0 +1,52 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_7
```

Comment:
i think we can up this to java 8

Reply:
we are still supporting Java 7 for now, though this may change, see #599